src/Symfony/Component/Mime/Email.php

```diff
@@ -442,7 +442,11 @@ private function generateBody(): AbstractPart
         }
 
         if ($attachmentParts) {
-            $part = new MixedPart($part, ...$attachmentParts);
+            if ($part) {
```

Comment:
ternary condition can be used wdyt?

Reply:
Personally I'm not a fan of ternary when both sides looks so similar.